### ISSUE TITLE
ZypMedia has been aquired

### ISSUE THREAD

**Issue by philipp-classen** [ghostery/trackerdb#119]
According to https://www.crunchbase.com/organization/extendtv, Sinclair Broadcast acquired ZypMedia in 2021.

Thus, its entry is outdated: https://github.com/ghostery/trackerdb/blob/main/db/patterns/zypmedia.eno

This requires some research (e.g how owns the company now and what is a current website).

**Comment by Switch0XD** [comment]
Hello @philipp-classen 
I did some research and found that ZypMedia Inc. is now part of Compulse Company. 
https://rocketreach.co/zypmedia-inc-now-part-of-compulse-profile_b5f33526f42d3468

**Comment by philipp-classen** [comment]
Yes, looks like there is chain from ZypMedia to Sinclair Broadcast Group. And it ended as Compulse (part of Sinclair Broadcast Group)

That means, a candidate for the new website could be: https://sbgi.net/marketing-services/

**Comment by philipp-classen** [comment]
Further questions are also under which domains Compulse is active. From what we could start to search for example pages that use it for analytics and define detection patterns.
